pkg/proxy/accesslog/log.go

```diff
@@ -101,18 +106,7 @@ func SetMetadata(md []string) {
 
 // Called with lock held.
 func logString(outStr string, retry bool) {
-	_, err := logBuf.WriteString(outStr + "\n")
-	if err != nil {
-		if retry {
-			log.WithError(err).WithField(FieldFilePath, logPath).Warn("Error encountered while writing to access log, retrying once...")
-
-			closeLogfileLocked()
-			openLogfileLocked(logPath)
-
-			// retry once
-			logString(outStr, false)
-		}
-	}
+	logger.Info(outStr)
```

Comment:
Why do we need to go through the logrus? Can't we just directly call into `Write()` of lumberjack? What happens to write errors?

Reply:
will do.. will be using Write() instead.